cypress/integration/myResources.spec.js

```diff
@@ -22,7 +22,7 @@ context('Actions', () => {
     cy.get('[data-testid=delete-confirm-dialog]').should('not.exist');
     cy.get('[data-testid=list-item-resources-123]').should('exist');
   });
-  it('can start deleting, then aborting deletion by clicking outside the confirm-delete dialog', () => {
+  it('can start deleting, then aborting deletion by clicking \n outside the confirm-delete dialog', () => {
```

Comment:
trenger ikke linebreak i teksten. Tenkte for spacing mellom linke 24 og 25 :-)

Reply:
Fikset